File: EducationApp.BusinessLogicLayer/Models/OrderItem/OrderItemModelItem.cs
using EducationApp.BusinessLogicLayer.Models.Base;
using static EducationApp.BusinessLogicLayer.Models.Enums.Enums;

namespace EducationApp.BusinessLogicLayer.Models.OrderItemModelItem
{
    public class OrderItemModelItem :BaseModel
    {
        public decimal Amount { get; set; }
        public CurrencyType Currency { get; set; }
        public long PrintingEditionId { get; set; }
        public int Count { get; set; }
        public string PrintingEditionType { get; set; }
        public string PrintingEditionName { get; set; } 
    }
}

File: ClientApp/src/app/shared/models/orderItem/orderItemModelItem.ts
import { CurrencyType } from 'src/app/shared/enums/CurrencyType';
import { BaseModel } from 'src/app/shared/models/Base/BaseModel';

export class OrderItemModelItem extends BaseModel {

    amount: number;
    currency: CurrencyType;
    printingEditionId: number;
    printingEditionType: string;
    count: number;
    printingEditionName: string;
    printingEditionPrice: number;

}